lib/travis/model/build.rb

```diff
@@ -166,23 +166,14 @@ def per_page
 
   after_save do
     unless cached_matrix_ids
-      update_column(:cached_matrix_ids, to_postgres_array(matrix_ids))
+      update_column(:cached_matrix_ids, matrix_ids)
     end
   end
 
   def state
     (super || :created).to_sym
   end
 
-  # AR 3.2 does not handle pg arrays and the plugins supporting them
-  # do not work well with jdbc drivers
-  # TODO: remove this once we're on >= 4.0
-  def cached_matrix_ids
-    if (value = super) && value =~ /^{/
-      value.gsub(/^{|}$/, '').split(',').map(&:to_i)
-    end
-  end
-
```

Comment:
😻

Reply:
love how the note `remove this once we're on >= 4.0` actually made sense for once